src/robusta/core/sinks/robusta/prometheus_discovery_utils.py

```diff
@@ -53,11 +53,11 @@ def _get_query_prometheus_value(self, query: str) -> Optional[float]:
             global_config = self.__global_config
             prometheus_params = PrometheusParams(**global_config)
             query_result = run_prometheus_query(prometheus_params=prometheus_params, query=query)
-            if query_result.result_type == "error" or query_result.vector_result is None:
+            if query_result.result_type == "error" or not query_result.vector_result:
                 logging.error(f"PrometheusDiscoveryUtils failed to get prometheus results.")
```

Comment:
f-string in this log.exception is redundant as well

Reply:
fixed